about.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="./styles/about.css">
+    <title>VueJs | Amsterdam</title>
+</head>
+<body>
+    <nav class="nav-header">
+        <i class="fa fa-facebook nav-icons" aria-hidden="true"></i>
+        <i class="fa fa-twitter nav-icons" aria-hidden="true"></i>
+        <ul class="nav-header-list">
+            <li>English</li>
+            <li>My Page</li>
+            <li>Logout</li>
+        </ul>
+    </nav>
+    <header class="header">
+        <img src="./images/logs.png" alt="" class="logo">
+        <ul class="nav-header-list">
+            <li><a href="./about.html">About</a></li>
+            <li><a href="./index.html">Program</a></li>
+            <li><a href="./index.html">Join</a></li>
+            <li><a href="./index.html">Sponsor</a></li>
+            <li><a href="./index.html">News</a></li>
+            <li><button class="btn-header">Join us in Amsterdam</button></li>
+        </ul>
+    </header>
+    <section class="mob-nav-bar">
+        <i class="fa fa-bars fa-1x" id='menu-btn' aria-hidden="true"></i>
+    </section>
+        <header class="h-desktop" id='menu'>
+            <i class="fa fa-times fa-2x" id='close-btn' aria-hidden="true"></i>
+            <ul class="nav-element">
+                <li><a class="nav-link" href="./index.html">Home</a></li>
+                <li><a class="nav-link" href="#">Program</a></li>
+                <li><a class="nav-link" href="#">News</a></li>
+            </ul>
+        
+        </header>
+    <section class="intro">
+        <div class="intro-content">
+            <h1>"VueJs Amsterdam"</h1>
+            <p class="intro-title">
+                WORLD'S MOST
+                SPECIAL AND
+                LARGEST VUE CONFERENCE
+            </p>
+            <section class="intro-desc">
+                Attend the Vue.js Community Event of the year and celebrate with Vue Communities
+            </section>
+            <span class="intro-contact">Please contact us per email for further questions about VueJs Amsterdam
+                <br><a href="">vue@amsterdam.com</a>
+            </span>
+        </div>
+    </section>
+    <section class="summit-logo">
+        <div class="program-title">
+            <h1>VueJs Amsterdam 2022</h1>
+            <span class="program-line"></span>
+        </div>
+        <div class="logo-desc">
+            <p>The logo of VueJs Amsterdam 2022 was decided through logo competition
+                from 8.June to 7.July</p>
+            <div class="logo">
+                <img src="./images/logo.png" alt="">
+            </div>
```

Comment:
- [ ] Please kindly place an appropriate `alt` attribute on all `img` tags. do this everywhere else you have `img` tags